PLEP-0008.rst

```diff
@@ -0,0 +1,113 @@
+===============================
+
++-------------------+---------------------------------------------+
+| PLEP              | number                                      |
++===================+=============================================+
+| title             | PlasmaPy Versioning and Releases            |
++-------------------+---------------------------------------------+
+| author(s)         | Nick Murphy                                 |
++-------------------+---------------------------------------------+
+| contact email     | namurphy@cfa.harvard.edu                    |
++-------------------+---------------------------------------------+
+| date created      | 2022-11-22                                  |
++-------------------+---------------------------------------------+
+| date last revised | 2022-11-22                                  |
++-------------------+---------------------------------------------+
+| type              | process                                     |
++-------------------+---------------------------------------------+
+| status            | discussion                                  |
++-------------------+---------------------------------------------+
+| DOI               |                                             |
+|                   |                                             |
++-------------------+---------------------------------------------+
+
+Brief Description
+-----------------
+
+PlasmaPy will use `calendar versioning`_ starting with its first
+release in 2023.
+
+This PLEP supersedes PLEP 5, which is no longer in effect.
+
+Tradeoff Analysis
+-----------------
+
+The advantages of calendar versioning are:
+
+* We can understand the age of a release solely from the version
+  number.
+
+* The quantity of changes to PlasmaPy is roughly proportional to the
+  time between releases.
+
+* The versions would be more consistent with projects like xarray,
+  dask, pip, Ubuntu, and PyCharm, which each use some form of calendar
+  versioning.
+
+* The differences between the last two releases beginning with ``0.``
+  are likely to be comparable to the differences between the last
+  ``0.`` release and ``1.0.0``. Hence, a change to ``1.0.0`` would
+  indicate more changes than were actually happening.
+
+The disadvantages to adopting calendar versioning are:
+
+* A change to calendar versioning would likely need to be permanent in
+  order to preserve monotonicity.
+
+* Calendar versioning is not used by many of the packages we depend on,
+  like NumPy, SciPy, matplotlib, and Astropy.
+
+* The version number itself would not indicate how mature the package
+  is.
+
+* Calendar-based version numbers of PlasmaPy would differ
+  substantially from the version numbers of prior releases.
+
+Versioning Specification
+------------------------
+
+PlasmaPy final releases will use `calendar based versioning`_ of the
+form ``YY.0M.MICRO``, where:
+
+* The major release number ``YY`` is the year minus 2000,
```

Comment:
Would this be better with a two-digit year or a four-digit year?

Reply:
4-digit YYYY, for the same reasons as https://github.com/dask/community/issues/100#issuecomment-704499489 :)